TUNING.md

```diff
@@ -0,0 +1,153 @@
+# Tuning Cppcheck Analysis
+
+There are several ways you can potentially improve the runtime of a Cppcheck analysis.
+
+Note: Most of these suggestions highly depend on your code base so they need to be evaluated before being applied. They might also depend the code of Cppcheck itself and your environment. So you might want to evaluate these on a regular basis.
+
+## Collecting Information
+
+To see the time each file took to analyze just specify the `--showtime=file-total` CLI option.
+
+## Tuning At Build-Level
+
+It is most likely that a pre-built binary is being used - either an official one or one provided by the packaging manager of my operating system.
```

Comment:
of your operating system

Note: I'm not seeing the "Suggest change" icon for some reason.